vizro-core/src/vizro/static/css/layout.css

```diff
@@ -175,3 +176,24 @@ div.dashboard_container .custom-tooltip {
 #page-container div[hidden] {
   display: none;
 }
+
+.collapsable-div {
+  background-color: var(--surfaces-bg-02);
+  height: 100%;
+  padding-top: 44px;
+  width: 12px;
+  padding-right: 12px;
+}
+
+#collapse_icon.material-symbols-outlined {
+  z-index: 1;
+  position: absolute;
```

Comment:
This also shouldn't be required anymore

```suggestion
```

Reply:
z-index is removed, but I need position property for positioning the icon when navigation panel is collapsed.